components/kyma-metrics-collector/pkg/process/process.go

```diff
@@ -360,8 +379,18 @@ func (p *Process) populateCacheAndQueue(runtimes *kebruntime.RuntimesPage) {
 		} else {
```

Comment:
how about removing this else block by continuing at the end of its 'if'

Reply:
removed